app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+
+        if len(cars) == 1:
+            income += self.calculate_washing_price(cars[0])
+            self.wash_single_car(cars[0])
+
+        else:
+            for car in cars:
+                if car.clean_mark < self.clean_power:
+                    income += self.calculate_washing_price(car)
+                    car.clean_mark = self.clean_power
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        diff = self.clean_power - car.clean_mark
+        if diff < 0:
+            return 0
```

Comment:
no need to make this check here, car.clean_mark always will be less then self.clean_power, u check this in serve_cars method using if condition